include/SSVOpenHexagon/Data/StyleData.hpp

```diff
@@ -48,6 +48,7 @@ class StyleData
     float _3dPerspectiveMult;
     float bgTileRadius{4500.f};
     unsigned int BGColorOffset{0};
+    float BGRotOff{0};
```

Comment:
Degrees? Radians? Pixels? A comment would be nice :)

Reply:
Degrees in lua, they get converted in the code to Radians to fit the calculations.